gdal/frmts/jpegxr/jpegxrdataset.cpp

```diff
@@ -215,12 +215,23 @@ struct JPEGXRDecoder final
         if (nErr != WMP_errSuccess || !pImageDecode)
             throw JPEGXRException(nErr);
 
+        PKPixelInfo pi;
+        GetPixelInfo(pi);
+
+        // Alpha
+        // 0: Decode without alpha channel
+        // 1: Decode only alpha channel (TODO: unused, might be controlled with driver option)
+        // 2: Decode image & alpha (default)
+        if (!!(pi.grBit & PK_pixfmtHasAlpha))
```

Comment:
Huh... If that's in a zone of constant values, that's indeed an issue. If that's on a few isolated pixels at the edge between a zone at 0 and another one at 255 in a lossy compressed image, that could be OK